File: submissions/sm_034_Venkat/week_3/day_3/session_2/count_consonants.js
function consonants(letter) {
    var letter = letter || 'abcdEfg';
    count= 0;
    for (i = 0; i < letter.length; i++) {
        if (letter[i] != 'a' && letter[i] != 'e' && letter[i] != 'i' && letter[i] != 'o' && letter[i] != 'u' &&
            letter[i] != 'A' && letter[i] != 'E' && letter[i] != 'I' && letter[i] != 'O' && letter[i] != 'U') 
            {
                console.log(letter[i]);
                count++
            }
        }
        console.log(count)
    }
    consonants('beNgalURu')

File: submissions/sm_026_rohit-kumar/week_13/day_4/session_1/count_consonants.py
lst = ['abcd', 'samsom', 'allahabad', 'bangalore']

consonants = ['b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'q', 'r', 's', 't', 'v', 'w', 'x', 'y', 'z']

count = 0
for str in lst:
    for s in str:
        if s.lower() in consonants:
            count = count + 1
    print(str, ": ", count)
    count = 0